### PR TITLE
Update configuration.asciidoc

### PR DESCRIPTION
**Pull request by leahleahy**

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -182,8 +182,8 @@ You must use the query bar to filter for a specific environment in versions prio
 | None       | Log level
 |============
 
-Fallback configuration setting to control the logging level for the agent.
-This setting has effect for the sink when sink specific option is not set explicitly.
+The fallback configuration setting to control the logging level for the {agent}.
+This setting occurs when the sink specific option is not set explicitly.
```
Comment (leahleahy) [inline docs/configuration.asciidoc:186]: Let me know if this rephrase works. I think I got it but not totally confident.

**Comment by SergeyKleyman** [comment]
@leahleahy Thank you for your contribution - could you please sign the Contributor License Agreement as described in [Submitting your changes section of Contributing to the Elastic APM PHP agent](https://github.com/elastic/apm-agent-php/blob/master/CONTRIBUTING.md#submitting-your-changes)?

File: docs/configuration.asciidoc [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/php[elastic.co].
 endif::[]
 
 [[configuration]]
 == Configuration
 
-You can utilize configuration options to adapt the Elastic APM agent to your needs.
-One way to do this is to configure settings via the `php.ini` file:
+Utilize configuration options to adapt the Elastic APM agent to your needs.
+Configure settings via the `php.ini` file:
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:10]:
```suggestion
One way to configure settings is with the `php.ini` file:```

File: docs/configuration.asciidoc [hunk 3]
```diff
@@ -64,17 +64,17 @@ INFO
 DEBUG
 TRACE
 ----
-For example, if you specify `WARNING` log level, only log records with levels `WARNING`, `ERROR` and `CRITICAL`
+For example, if you specify `WARNING` log level, only log records with levels `WARNING`, `ERROR`, and `CRITICAL`
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:67]:
```suggestion
For example, if you specify a `WARNING` log level, only log records with levels `WARNING`, `ERROR`, and `CRITICAL`
```

File: docs/configuration.asciidoc [hunk 4]
```diff
@@ -64,17 +64,17 @@ INFO
...
 will be emitted.
 
 `OFF` is only used to disable agent logging.
-There cannot be log records with level `OFF`.
+Log records cannot use `OFF`.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:71]:
```suggestion
```

File: docs/configuration.asciidoc [hunk 5]
```diff
@@ -64,17 +64,17 @@ INFO
...
 
-The agent supports logging to the following sinks: syslog and stderr.
-You can control the level of logging for individual sinks with the
+The {agent} supports logging to the following sinks: syslog and stderr.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:73]:
```suggestion
The agent supports logging to the following sinks: syslog and stderr.
```

File: docs/configuration.asciidoc [hunk 6]
```diff
@@ -111,11 +111,11 @@ elastic_apm.log_level_syslog=TRACE
 | None       | String
 |============
 
-This string is used to ensure that only your agents can send data to your APM Server.
-You must have created the API key using the APM Server's {apm-server-ref-v}/api-key.html[command line tool].
+This string is used to ensure that only your {agent}s can send data to your APM Server.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:114]:
```suggestion
This string is used to ensure that only your agents can send data to your APM Server.
```

File: docs/configuration.asciidoc [hunk 7]
```diff
@@ -111,11 +111,11 @@ elastic_apm.log_level_syslog=TRACE
...
+You must have created the API key using the APM Servers {apm-server-ref-v}/api-key.html[command line tool].
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:115]:
```suggestion
You must have created the API key using the APM Server {apm-server-ref-v}/api-key.html[command line tool].
```

File: docs/configuration.asciidoc [hunk 8]
```diff
@@ -139,7 +139,7 @@ your communications using HTTPS. Unless you do so, your API Key could be observe
 | false       | Boolean
 |============
 
-Setting to false will completely disable the agent.
+Setting to false will completely disable the {agent}.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:142]:
```suggestion
Setting to false will completely disable the agent.
```

File: docs/configuration.asciidoc (hunk 1 above)
Comment (bmorelli25) [inline docs/configuration.asciidoc:186]:
```suggestion
A fallback configuration setting to control the logging level for the agent.
Only used when a sink-specific option is not explicitly set.
```

File: docs/configuration.asciidoc [hunk 9]
```diff
@@ -240,8 +240,8 @@ See <<configure-logging>> for details.
 | None       | String
 |============
 
-This string is used to ensure that only your agents can send data to your APM Server.
-Both the agents and the APM Server have to be configured with the same secret token.
+This string is used to ensure that only your {agent}s can send data to your APM Server.
+Both the {agent}s and the APM Server have to be configured with the same secret token.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:244]:
```suggestion
This string is used to ensure that only your agents can send data to your APM Server.
Both the agents and the APM Server have to be configured with the same secret token.
```

File: docs/configuration.asciidoc [hunk 10]
```diff
@@ -272,15 +272,15 @@ your communications using HTTPS. Unless you do so, your secret token could be ob
...
+the request is canceled and the events are discarded.
 
 This configuration option supports the duration suffixes: `ms`, `s` and `m`.
 For example: `10s`.
 This option's default unit is `s`, so `5` is interpreted as `5s`.
 
 If the value is `0` (or `0ms`, `0s`, etc.) the timeout for sending events to the APM Server is disabled.
 
-If the value is negative, it's invalid, and the default value is used instead.
+If the value is negative and invalid, the default value is used instead.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:283]:
```suggestion
Negative values are invalid and result in the default value being used instead.
```

File: docs/configuration.asciidoc [hunk 11]
```diff
@@ -317,12 +317,12 @@ The URL for your APM Server. The URL must be fully qualified, including protocol
...
 
 This is used to keep all the errors and transactions of your service together
-and it is the primary filter in the Elastic APM user interface.
+and is the primary filter in the Elastic APM user interface.
 
 NOTE: The service name must conform to this regular expression: `^[a-zA-Z0-9 _-]+$`.
-In less regexy terms: your service name must only contain characters from the ASCII alphabet,
-numbers, dashes, underscores and spaces.
-Characters in service name which don't match regular expression will be replaced by `_` (underscore) character.
+Your service name must only contain characters from the ASCII alphabet,
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:323]:
```suggestion
In other words, a service name must only contain characters from the ASCII alphabet,
```

File: docs/configuration.asciidoc [hunk 12]
```diff
@@ -360,14 +360,14 @@ the output of git rev-parse HEAD.
...
-Limits the amount of spans that are recorded per transaction.
+This limits the amount of spans that are recorded per transaction.
 This is helpful in cases where a transaction creates a very high amount of spans,
 for example, thousands of SQL queries.
 Setting an upper limit helps prevent overloading the Agent and APM server in these edge cases.
 
 If the value is `0` no spans will be collected.
 
-If the value is negative, it's invalid, and the default value is used instead.
+If the value is negative and invalid, the default value is used.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:370]:
```suggestion
Negative values are invalid and result in the default value being used instead.
```

File: docs/configuration.asciidoc [hunk 13]
```diff
@@ -386,9 +386,9 @@ If the value is negative, it's invalid, and the default value is used instead.
 |============
 
 By default, the agent will sample every transaction (e.g., a request to your service).
-To reduce overhead and storage requirements, you can set the sample rate to a value between 0.0 and 1.0.
-The agent will still record the overall time and result for unsampled transactions,
-but not context information, labels, or spans.
+To reduce overhead and storage requirements, set the sample rate to a value between `0.0` and `1.0`.
+The agent still records the overall time and result for unsampled transactions,
+but not for context information, labels, or spans.
```
Comment (bmorelli25) [inline docs/configuration.asciidoc:391]:
```suggestion
but not context information, labels, or spans.
```

**Comment by bmorelli25** [comment]
Jenkins, run the tests please

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Review by bmorelli25 — APPROVED** [review]
Pending doc build, LGTM

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**